prac_06/guitars.py

```diff
@@ -0,0 +1,34 @@
+"""
+CP1404/CP5632 Practical
+Estimated time: 60min
+Playing the Guitars (not really)
+"""
+
+from prac_06.guitar import Guitar
+
+
+def main():
+    """A program to store and display guitars"""
+
+    guitars = []
+    print("My guitars!")
+    name = input("Name: ")
+    while name != "":
+        year = int(input("Year: "))
+        cost = float(input("Cost: $"))
+        guitar_to_add = Guitar(name, year, cost)
+        guitars.append(guitar_to_add)
+        print(f"{guitar_to_add}, added")
+        name = input("Name: ")
+
+    guitars.append(Guitar("Gibson L-5 CES", 1922, 16035.40))
+    guitars.append(Guitar("Line 6 JTV-59", 2010, 1512.9))
+    guitars.append(Guitar("Fender Stratocaster", 2014, 765.40))
+
+    print("These are my guitars:")
+    for i, guitar in enumerate(guitars, 1):
+        vintage_string = "(vintage)" if guitar.is_vintage() else ""
+        print(f"Guitar {i}: {guitar.name:>20} ({guitar.year}), worth ${guitar.cost:10,.2f} {vintage_string}")
```

Comment:
The prac 6 document's sample output shows that the output is sorted on year. I think this code prints guitars in the order they are entered.